src/Http/Http.Features/test/Microsoft.AspNetCore.Http.Features.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
This package stayed on netstandard, presumably the tests should still cover net461?

Reply:
Updated.